File: edX/CS50x/Week 1/Problem Set 1/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int n;
    n = get_int("Height: ");
    while (n < 1 || n > 8)
    {
        n = get_int("Height: ");
    }

    int k = 1;
    for (int x = 1; x <= n; x++)
    {
        for (int y = x; y <= n-1; y++)
        {
            printf(" ");
        }

        for (int j = 1; j <= k; j++)
        {
            printf("#");
        }
        k += 1;

        printf("\n");
    }
}

File: edX/CS50x/Week 6/Problem Set 6/sentimental-mario-more/mario.py
from cs50 import get_int

num = get_int("Height: ")
while True:
    if num < 1 or num > 8:
        num = get_int("Height: ")
    else:
        break

k = 1
x = 1
while x <= num:
    y = x
    while y <= num-1:
        print(end=" ")
        y += 1

    j = 1
    while j <= k:
        print("#", end="")
        j += 1

    print(end="  ")

    a = 1
    while a <= k:
        print("#", end="")
        a += 1

    k += 1
    x += 1

    print()